### ISSUE TITLE
F# streams support

### ISSUE THREAD

**Issue by aprooks** [yevhen/Orleankka#73]
That's what I tried

https://gist.github.com/aprooks/3f9cb68a3f17ac041362

the output is 
```
Started system
recieved System.String
handled
recieved Program+TestRecord
handled record
```

so streams are only working if Orleankka.Actor is used (not Fsharp adapter) and DU's are completely ignored

**Comment by aprooks** [comment]
@AntyaDev ping;)

**Comment by AntyaDev** [comment]
@aprooks good catch, thanks!

**Comment by yevhen** [comment]
This was recently fixed (with 0.14.0). Just declare actor as `inherit Actor<obj>()` . See example [here](https://github.com/yevhen/Orleankka/blob/a639f348a1bca8b69014ad4185058427ddef4f44/Source/FSharp.Api.Tests/RequestResponse.fs#L21)

**Comment by AntyaDev** [comment]
@aprooks @yevhen also you should specify '*' in Filter to handle all incoming messages.
```
[StreamSubscription(Source = "sms:select-all", Target = "#", Filter = "*")]
type MyActor()
    inherit Actor<obj>()

```
